internal/services/search/search_service_resource_test.go

```diff
@@ -278,7 +278,7 @@ func TestAccSearchService_partitionCountInvalidBySku(t *testing.T) {
 		{
 			Config:      r.partitionCount(data, "basic", 3),
```

Comment:
should we change this to 4 since 3 is now a valid partition count?

Reply:
Yes will update and the initial count I mentioned it to 4 also I have tested both cases successfully with local provider 
1. created resource with basic SKU partition count 3
2. with partition count 4 I got error message  "Error: 'partition_count' values greater than 3 cannot be set for the "basic" SKU, got 4)"